sefaria/helper/crm/salesforce.py

```diff
@@ -120,8 +123,18 @@ def find_crm_id(self, email=None):
         except:
             return False
 
-    def subscribe_to_lists(self, email, first_name=None, last_name=None, lang="en", educator=False):
-        # TODO: Implement once endpoint exists
+    def subscribe_to_lists(
+        self, 
+        email: str, 
+        first_name: Optional[str] = None, 
+        last_name: Optional[str] = None, 
+        lang: str = "en", 
+        educator: bool = False,
+        mailing_lists: Optional[List[str]] = None) -> Any:
+
+        if mailing_lists is None:
```

Comment:
Can be more simply written as `mailing_lists = mailing_lists or []`. This is fairly common as a first line in a function given Python's buggy behavior with lists as default parameters.